contracts/SponsorFaucet.sol

```diff
@@ -31,12 +31,13 @@ contract SponsorFaucet is Ownable, Pausable, ReentrancyGuard {
     /*** Dapp dev calls ***/ 
     //apply cfx for gas & storage
     function applyFor(address dapp, uint256 upper_bound) public nonReentrant whenNotPaused {
```

Comment:
It's up to `owner` to set the `upper_bound` of transaction fee. Otherwise, applicant could specify a very large value.

Reply:
fixed, upper_bound is colltrolled by owner now